### ISSUE TITLE
No Await Expression for  Websocket Closing

### ISSUE THREAD

**Issue by wes8ty** [permitio/fastapi_websocket_rpc#36]
Was testing this library out and the following line `self._websocket.close(code)` needs an `await`

``` py
async def close(self, code: int = 1000):
    if self._websocket is not None:
        # Case opened, we have something to close.
        self._websocket.close(code)
```

**Comment by orweis** [comment]
Want to open a PR? :)

**Comment by mberdyshev** [comment]
Should it be closed now by #43?

**Comment by orweis** [comment]
Yes you are correct - @mberdyshev - thank you.

Closed by https://github.com/permitio/fastapi_websocket_rpc/pull/43
